r/src/R/ai/h2o/sparkling/SharedBackendConf.R

```diff
@@ -330,9 +335,19 @@ SharedBackendConf <- setRefClass("SharedBackendConf", methods = list(
 
     setVerifySslCertificates = function(verify) { invoke(jconf, "setVerifySslCertificates", verify); .self },
 
-    setHiveSupportEnabled = function() { invoke(jconf, "setHiveSupportEnabled"); .self },
+    setKerberizedHiveEnabled = function() { invoke(jconf, "setKerberizedHiveEnabled"); .self },
+
+    setHiveSupportEnabled = function() {
+      warning("The method 'setHiveSupportEnabled' is deprecated and will be removed in 3.32. Use 'setKerberizedHiveEnabled' instead!")
+      invoke(jconf, "setHiveSupportEnabled"); .self
```

Comment:
nit: shouldn't  `setKerberizedHiveEnabled` be called here to be consistent with `isKerberizedHiveEnabled`

Reply:
Yup